blueflood-core/src/main/java/com/rackspacecloud/blueflood/io/astyanax/AstyanaxIO.java

```diff
@@ -116,7 +116,7 @@ private static ConnectionPoolConfigurationImpl createPreferredConnectionPoolConf
         return connectionPoolConfiguration;
     }
 
-    protected static Keyspace getKeyspace() {
+    public static Keyspace getKeyspace() {
```

Comment:
I vote for removing the tests: writeDatastaxReadOldAstyanax() and writeOldAstyanaxReadDatastax(), because those 2 tests are not much different than writeDatastaxReadAstyanax() and writeAstyanaxReadDatastax(). Eventually they will call the methods in AstyanaxWriter.insertLocator(), yes? Or am I completely wrong on that?